### DISCUSSION TITLE
文本分类，单字情况怎么分类？有四个方向，而paddleocr只分了0和180度

### DISCUSSION THREAD

**Discussion by cqray1990** [PaddlePaddle/PaddleOCR#13727] [Q&A]
文本分类，单字情况怎么分类？有四个方向，而paddleocr只分了0和180度

**Comment by GreatV** [comment]
可以考虑使用 [PaddleClas](https://github.com/PaddlePaddle/PaddleClas) 或者 [paddlex](https://github.com/paddlepaddle/paddlex) 自己训练一个模型来分类。
